REVIEWME.txt

```diff
@@ -0,0 +1 @@
+Please review me!
```

Comment:
The implementation of comb filter and delay buffer logic looks pretty clean overall. One thing to notice and be careful about is that the read audio from hound has dimension [channel_size][sample_length] but the handling took it reversed, it might cause some problems later on. 
I like how the edge cases are considered and handled in main with proper error messages reported. The test cases looks pretty well implemented on generating different kinds of output (which confused me a lot when I was implementing them).

Reply:
One more thing I think you need to be more careful about is the size of delay buffer and input, it seems like you initialize one for every channel, it is ok in your implementation since you did not divide the input into multiple windows/chunks (which is a pretty typical process for audio processing) and processed it in one loop, but when it comes to industrial standards, you might want to consider handle the continuous delay between each chunk with proper splitting methods since processing the whole audio at once might not be practical for a lot of its real-life scenarios.